images/cert-manager/image.yaml

```diff
@@ -0,0 +1,3 @@
+versions:
+  - apko:
+      config: configs/latest.apko.yaml
```

Comment:
This is going to mess with the post-submit "policy verification" logic.

IMO we should file an issue to bring back some form of policy verification, and then just rip it out for now.

The place we're (currently) doing it is ~wrong anyways because it's after we've tagged stuff, so it won't block stuff like catalog syncing.

Reply:
https://github.com/chainguard-images/images/issues/941